lib/common/ipc_attrd.c

```diff
@@ -143,7 +143,8 @@ create_attrd_op(const char *user_name)
     xmlNode *attrd_op = create_xml_node(NULL, __func__);
 
     crm_xml_add(attrd_op, PCMK__XA_T, PCMK__VALUE_ATTRD);
-    crm_xml_add(attrd_op, PCMK__XA_SRC, pcmk__s(crm_system_name, "unknown"));
+    crm_xml_add(attrd_op, PCMK__XA_SRC,
+                pcmk__s(crm_system_name, PCMK_VALUE_UNKNOWN));
```

Comment:
ditto

Reply:
Yep also a poor fit. `crm_system_name` is sort of enumerated in practice internally, but `crm_log_preinit()` and `crm_log_init()` are public.